src/databricks/labs/ucx/source_code/base.py

```diff
@@ -97,10 +98,12 @@ class CurrentSessionState:
     This class can be used to track various aspects of a session, such as the current schema.
 
     Attributes:
+        catalog (str): The current schema of the session. If not provided, it defaults to 'DEFAULT_CATALOG'.
```

Comment:
Make the docsrting an f-string and insert the `{DEFAULT_CATALOG}` variable

Reply:
Same for DEFAULT_SCHEMA

Reply:
docstring cannot be f-string

Reply:
why not?

Reply:
it is in the language spec :) 
https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals:~:text=Formatted%20string%20literals%20cannot%20be%20used%20as%20docstrings%2C%20even%20if%20they%20do%20not%20include%20expressions.

Reply:
Ah, interesting, learned something new here :) I am pretty sure I used it one time, but probably missed that the `__doc__` is set to None without failure 🥴